pkg/helpers/queuekey.go

```diff
@@ -105,6 +105,31 @@ func ClusterManagerDeploymentQueueKeyFunc(clusterManagerLister operatorlister.Cl
 	}
 }
 
+func BootstrapSecretQueueKeyFunc(klusterletLister operatorlister.KlusterletLister) factory.ObjectQueueKeyFunc {
```

Comment:
i would put this in the controller since it seems only  bootstrap controller use it only.

Reply:
done